### ISSUE TITLE
Segfault when feeding an multi digraph to GeneratorsOfEndomorphismMonoid

### ISSUE THREAD

**Issue by wilfwilson** [digraphs/Digraphs#367]
```
gap> D := Digraph([[2,2],[]]);
<immutable multidigraph with 2 vertices, 2 edges>
gap> GeneratorsOfEndomorphismMonoid(D);
/usr/local/bin/gap: line 3: 58880 Segmentation fault: 11
```
Presumably `GeneratorsOfEndomorphismMonoid` should not accept multidigraphs.

**Comment by james-d-mitchell** [comment]
This is bad, I'm going to assign myself, and try to fix this soon!

**Comment by james-d-mitchell** [comment]
Resolved in v1.4.0
